server/src/test/java/org/elasticsearch/cluster/coordination/RareClusterStateIT.java

```diff
@@ -370,11 +376,9 @@ public void testDelayedMappingPropagationOnReplica() throws Exception {
 
         // Now make sure the indexing request finishes successfully
         disruption.stopDisrupting();
-        assertBusy(() -> {
```

Comment:
These busy assert made no sense, we're never resending the requests we're asserting on here, all these do is make a failed test run take an extra `10s`.